Skype/WebSDK/docs/PTVideoGroup.md

```diff
@@ -6,83 +6,113 @@
+            person.video.state.when('Connected', function () {
+                // video is availabe ... lets assign a container.
 
-    listeners.push(conversation.selfParticipant.video.state.when('Connected', function () {
-        // set up local video container
-        setupContainer(conversation.selfParticipant, 'large', content.querySelector('.selfVideoContainer'));
+                // formats include: 'Stretch', 'Fit' and 'Crop'
+                person.video.channels(0).stream.source.sink.format('Stretch');
+                person.video.channels(0).stream.source.sink.container(/* DOM node such as a DIV */);
 
-        listeners.push(conversation.participants.added(function (person) {
-            // person.displayName() has joined the conversation
-            listeners.push(person.video.state.when('Connected', function () {
-                // set up remote video container
-                if (Object.keys(videoMap).length === 1) {
-                    videoMap[person.displayName()] = 2;
-                    setupContainer(person, 'large', content.querySelector('.remoteVideoContainer2'));
-                }
-                else {
-                    videoMap[person.displayName()] = 1;
-                    setupContainer(person, 'large', content.querySelector('.remoteVideoContainer1'));
-                }
+                // now we can render it
                 person.video.channels(0).isStarted(true);
-            }));
-        }));
-    }));
-
-    listeners.push(conversation.state.changed(function (newValue, reason, oldValue) {
-        if (newValue === 'Disconnected' && (oldValue === 'Connected' || oldValue === 'Connecting')) {
-            // Conversation ended
-        }
-    }));
-
-    conversation.participants.add(id);
-    conversation.participants.add(id2);
-    conversation.videoService.start().then(null, function (error) {
-        // handle error
-    });
-  ```
 
-2. **Advanced**: Track remote participant video state
+                // NOTE: .isStarted() only needs to be called for remote participants in group conversations
+                // it dictates wether or not the participant's video should be rendered
+            });
+        });
+    ```
+
+1. Last but not least we can add participants to the conversation and start the video conversation.
 
     ```js
-    listeners.push(person.video.channels(0).isVideoOn.when(true, function () {
-        // person.displayName() started streaming their video
-    }));
-    listeners.push(person.video.channels(0).isVideoOn.when(false, function () {
-       // person.displayName() stopped streaming their video
-    }));
-  ```
+        conversation.participants.add('<participant1.id>');
+        conversation.participants.add('<participant2.id>');
+
+        // starting the video conference
+        conversation.videoService.start().then(null, function (error) {
+            // handle error
+        });
+    ```
 
-3. End the conversation
+## Ending the conversation
```

Comment:
I'd also include the videoService disconnected listener here that you removed from the above snippet